loadbalancer_test.go

```diff
@@ -19,19 +20,19 @@ func setupTestClient() (*Client, *http.ServeMux) {
 	server := httptest.NewServer(mux)
 	config := Config{
 		APIUrl:     server.URL,
-		UserUUID:   "uuid",
-		APIToken:   "token",
+		UserUUID:   os.Getenv("USER_UUID"),
+		APIToken:   os.Getenv("API_TOKEN"),
 		HTTPClient: http.DefaultClient,
```

Comment:
setupTestClient function should be moved to file `testcommon.go`.
No need ` os.Getenv("USER_UUID")` `os.Getenv("API_TOKEN")`, at the end it's dummy values to configure client.

Reply:
A further improvement, setupTestClient function should return `server`, so we can use `defer  server.Close()` directly after calling `setupTestClient` in the tests.